tslearn/docs/user_guide/matrix-profile.rst

```diff
@@ -0,0 +1,48 @@
+.. _matrix-profile:
+
+Matrix Profile
+==============
+
+The Matrix Profile, :math:`MP`, is a new time series that can be calculated based on an input time series :math:`T` and a subsequence length :math:`m`. :math:`MP_i` corresponds to the minimal distance from the query subsequence :math:`T_{i\rightarrow i+m}` to :math:`T` [1]_.  As the distance from the query subsequence to itself will be equal to zero, :math:`T_{i-\frac{m}{4}\rightarrow i+\frac{m}{4}}` is considered as an exclusion zone. In order to construct the Matrix Profile, a distance profile which is similar to the distance calculation used to transform time series into their shapelet-transform space, is calculated for each subsequence, as illustrated below:
```

Comment:
> corresponds to the minimal distance from the query subsequence :math:`T_{i\rightarrow i+m}` to :math:`T`

Maybe change to

> corresponds to the minimal distance from the query subsequence :math:`T_{i\rightarrow i+m}` to any subsequence in :math:`T`

to make things clearer?

Reply:
Also, in the last sentence of the line, maybe you could add a link to the corresponding section in the shapelet user guide?